stdlib/src/builtin/none.mojo

```diff
@@ -37,3 +37,28 @@ struct NoneType(CollectionElement):
             other: Another `NoneType` instance to copy.
         """
         self._value = None
+
+    @no_inline
+    fn __str__(self) -> String:
+        """Returns the string representation of `None`.
+
+        Returns: `"None"`.
+        """
+        return "None"
+
+    @no_inline
+    fn __repr__(self) -> String:
+        """Return the string representation of `None`.
+
+        Returns: `"None"`.
+        """
+        return "None"
+
+    @no_inline
+    fn format_to(self, inout writer: Formatter):
```

Comment:
**Suggestion** Can we write a test for this?  I think this will currently fail still in `test/utils/test_format.mojo`:

```
ef test_formatter_of_none():
   var x = None
   var fmt = Formatter(x)
   fmt.write(x)
   assert_equal(x, "None")
```

I think it will fail to construct the formatter.